testing_and_setup/compass/ocean/dam_break/default/004m/config_init1.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0"?>
+<config case="init_step1">
+  <add_link source_path="mpas_tools" source="mesh_tools/periodic_hex/namelist.input" dest="namelist.input.default"/>
+  <add_link source_path="mpas_tools" source="mesh_tools/periodic_hex/periodic_grid" dest="periodic_hex"/>
+
+	<add_executable source="model" dest="ocean_model"/>
+	<add_executable source="cell_culler" dest="MpasCellCuller.x"/>
+	<add_executable source="mesh_converter" dest="MpasMeshConverter.x"/>
+
+	<namelist name="namelist.ocean" mode="init">
```

Comment:
@caozd999, Namelist and streams aren't needed in this code since it isn't in the run script.

Reply:
My mistake, got confused here.  Please ignore.